packages/app/src/index.ts

```diff
@@ -1,11 +1,67 @@
 
-import { AeroGearConfiguration, INSTANCE } from "@aerogear/core";
+import find from "lodash.find";
+import console from "loglevel";
+import { AeroGearConfiguration, ServiceConfiguration, MetricsService } from "@aerogear/core";
+
+/**
+ * Abstraction for AeroGear configuration
+ */
+export class ConfigurationService {
```

Comment:
can we move this to be part of the core module? This way this service module will not need to depend on the `app` module anymore, but just the `core` module. This way the `core` module is the only command module that is shared across all the service modules. All the services module will then be decoupled from the `app` module completely.